perception/autoware_multi_object_tracker/lib/tracker/model/tracker_base.cpp

```diff
@@ -192,6 +242,35 @@ bool Tracker::updateWithoutMeasurement(const rclcpp::Time & timestamp)
   return true;
 }
 
+bool Tracker::createPseudoMeasurement(
+  const types::DynamicObject & meas, types::DynamicObject & pred)
+{
+  // Apply linear fall‑off weight on dist square
+  const double dx = meas.pose.position.x - pred.pose.position.x;
+  const double dy = meas.pose.position.y - pred.pose.position.y;
+  const double dist2 = dx * dx + dy * dy;
+  constexpr double d_max_square_inv = 1 / 2.0;
```

Comment:
w_pose will saturate when dist2 becomes 2.0 (distance of 1.414 m) 
```suggestion
  constexpr double d_max_square_inv = 1 / 2.0; # saturate when distance overs 1.414m
```